user/reference/overview.md

```diff
@@ -240,7 +240,7 @@ If you need more than the default 2vCPU and RAM, you can run a build on a larger
 
 ```yaml
 vm:
-  size: [large|x-large|2x-large]
+  size: [medium|large|x-large|2x-large]
```

Comment:
actually if you use no `size` tag, the medium/standard is assumed by default and is not necessary to be explicitly called out (unless you want a full visibility in mixed size environment). Maybe there should be a note added about `medium` being assumed by default and thus not requiring explicit `sie: medium`?